Adding Asepct Example (#124)
MiniFate/MiniFate: _chapters/00-outline.md

  This establish a new fact in the game because aspects are always true.
  The new aspect can be invoked by paying additional fate points.

Players begin each session with fate points equal to their PC's {Level}.
The GM starts with fate points equal to the highest {Level} PC to spend on their own rolls.
The GM has an unlimited pool of fate points for compels.

> > The {BoatName} is impounded after they were caught with an illicit cargo of unstamped rice.

Reviewer: ```suggestion
> The {BoatName} is impounded after being caught with an illicit cargo of unstamped rice.
```

> > The {BoatName} is impounded after they were caught with an illicit cargo of unstamped rice.
> > The local magistrate believes this is a simple case of cheap traders, but he'll soon get word from the capital that there is more at stake.
> > That's when the navy will get involved.
> > Naka and Torbyorn have to act now if they want to escape without having to blast their way out.
> > Unfortunately, the magistrate's office is booked for weeks.

Reviewer: ```suggestion
Local authorities find Naka and Torbyorn with a cargo of unstamped rice. 
The {BoatName} is impounded, and it'll be weeks before their appeal can be heard. 
But they can't afford to stick around.
They need to speak to the magistrate *now*. 
Before she realizes who they are. 
If she calls in the navy, they'll be hopelessly outgunned.
```

Something about the wording isn't hitting me quite right. Here's another shot at it

>
> {NakaPlayer} has an idea.

> > He use a fate point to **Create an aspect**: `The Magistrate Knows, and Fears, My Father`;

Reviewer: "he" -> do we need `{NakaPlayerPronoun}` for consistency? I thought one of the players was female

Author: NakaPlayer is currently `Barry`. :-/ But yeah was (unfortunately) thinking the same. Could just move to `they` for everyone too...

Author: Pronouns are hard... Have to have one for he, him, his and also capitals... :nauseated_face:

> > He use a fate point to **Create an aspect**: `The Magistrate Knows, and Fears, My Father`;

Reviewer: `The Magistrate Owes my Father a Debt`?

Sets us up to make a promise, rather than just rolling to intimidate

Author: Yeah I like this (despite Naka being the proto-typical murder hobo...)
